client/src/components/Footer.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+import styled from "@emotion/styled";
+import AccountIcon from "../stories/assets/account-icon.svg";
+import ChatIcon from "../stories/assets/message-icon.svg";
+import SettingsIcon from "../stories/assets/settings-icon.svg";
+
+const MainFooter = styled.footer`
+  width: 100%;
+
+  background: #ade1cd;
+  border: none;
+  border-radius: 0;
```

Comment:
I think these props are not required for a footer. A footer shouldn't have a border by default.